src/juxt/site/alpha/handler.clj

```diff
@@ -2,20 +2,22 @@
 
 (ns juxt.site.alpha.handler
   (:require
+   [cheshire.core :as json]
    [clojure.instant :refer [read-instant-date]]
    [clojure.set :as set]
    [clojure.string :as str]
    [clojure.tools.logging :as log]
-   [crux.api :as x]
+   [crux.api :as crux]
    [crypto.password.bcrypt :as password]
    [juxt.apex.alpha.openapi :as openapi]
    [juxt.dave.alpha :as dave]
    [juxt.dave.alpha.methods :as dave.methods]
    [juxt.jinx.alpha.vocabularies.transformation :refer [transform-value]]
    [juxt.pass.alpha.authentication :as authn]
+   [juxt.pass.alpha.session :as session]
    [juxt.pass.alpha.pdp :as pdp]
    [juxt.pick.alpha.core :refer [rate-representation]]
-   [juxt.pick.alpha.ring :refer [pick decode-maybe]]
+   [juxt.pick.alpha.ring :refer [decode-maybe pick]]
```

Comment:
`pick` having same name as namespace `pick` which could be a bit confusing. Should we change name on either of the two?